release-notes/unreleased/IGAPP-794-remove-gradient-local-news.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-794
+show_in_stores: true
+platforms:
+  - web
+en: Removed weird color gradient from news.
```

Comment:
Why no German release note?

Reply:
There is no store for web so they are only ever posted/used in english.